_internals/system-views.md

```diff
@@ -4,47 +4,47 @@ title: Babelfish system view
 nav_order: 4
 ---
 
-## Babelfish system views 
+## Babelfish system views
 
-Once in a while, an application has to access an MS SQL-compatible system view to
+Once in a while, an application has to access an MS SQL-compatible system view
```

Comment:
Avoid colloquial language like 'once in a while'

Reply:
Replaced with "One in a blue moon".

No, kidding. I used "Sometimes".